src/main/java/anya/parser/Parser.java

```diff
@@ -0,0 +1,253 @@
+                            throw new InvalidArgumentException("☹ Waku waku! Please input in the following format:\n"
+                                    + "    event <taskName> /from <startTime> /to <endTime>");
+                        }
+
+                        String taskName = details.split("/from")[0].trim();
+                        String startTime = details.split("/from")[1].trim().split("/to")[0].trim();
+                        LocalDateTime startTimeDate = convertStringToDate(startTime);
+                        String endTime = details.split("/to")[1].trim();
+                        Task t = new Event(taskName, startTime, endTime);
+                        tasks.add(t);
+
+                        System.out.println("    Waku waku! I've added this task:\n" + t);
+                        System.out.println("    Now you have " + tasks.size() + " tasks in the list!");
+                        break;
+                    }
+                    case DELETE: {
+                        // Error: No argument or Multiple arguments provided
+                        if (details.isEmpty() || details.split(" ").length != 1) {
+                            throw new InvalidArgumentException("☹ Waku waku! Please input in the following format:\n"
+                                    + "    delete <taskNumber>");
+                        }
+                        // Error: Argument provided is not a number
+                        try {
+                            Integer.parseInt(details);
+                        } catch (NumberFormatException e) {
+                            throw new InvalidArgumentException("☹ Waku waku! "
+                                    + "Please only input INTEGERs after the word unmark!");
+                        }
+                        // Error: Argument provided is not within anya.task numbers
+                        int taskNumber = Integer.parseInt(details);
+                        if (taskNumber < 1 || taskNumber > tasks.size()) {
+                            throw new InvalidArgumentException("☹ Waku waku! "
+                                    + "I don't see a anya.task with the number:" + taskNumber);
+                        }
+
+                        Task t = tasks.get(taskNumber - 1);
+                        tasks.remove(t);
+
+                        String result = "    Waku waku! I've removed this task:\n" + t;
+                        result += "    Now you have " + tasks.size() + " tasks in the list!";
+                        System.out.println(result);
+                        break;
+                    }
+                    case FIND: {
+                        tasks.find(details);
+                        break;
+                    }
+                    default:
+                        String result = "☹ Waku waku!!! I'm sorry, but I don't know what that means (yet) :( ";
+                        System.out.println(result);
+                }
```

Comment:
Should there be indentation for the `case` clauses in a _switch_ statement? 

If you are using IntelliJ, perhaps you can open file settings, go to **Editor > CodeStyle > Java > Wrapping and Braces, at 'switch' statement, untick the box 'Indent case branches'** 👍 

Additional tip: To reformat a section of code in Intellij, can refer to this [guide](https://www.jetbrains.com/idea/guide/tips/reformat-file/#:~:text=Sometimes%20code%20formatting%20can%20get,L%20(Windows%2FLinux))!